In [1]:
from twitterscraper import query_tweets

list_of_tweets = query_tweets("Trump OR Clinton", 10)

INFO: queries: ['Trump OR Clinton since:2006-03-21 until:2006-11-13', 'Trump OR Clinton since:2006-11-13 until:2007-07-08', 'Trump OR Clinton since:2007-07-08 until:2008-03-02', 'Trump OR Clinton since:2008-03-02 until:2008-10-25', 'Trump OR Clinton since:2008-10-25 until:2009-06-19', 'Trump OR Clinton since:2009-06-19 until:2010-02-12', 'Trump OR Clinton since:2010-02-12 until:2010-10-07', 'Trump OR Clinton since:2010-10-07 until:2011-06-01', 'Trump OR Clinton since:2011-06-01 until:2012-01-25', 'Trump OR Clinton since:2012-01-25 until:2012-09-18', 'Trump OR Clinton since:2012-09-18 until:2013-05-13', 'Trump OR Clinton since:2013-05-13 until:2014-01-06', 'Trump OR Clinton since:2014-01-06 until:2014-08-31', 'Trump OR Clinton since:2014-08-31 until:2015-04-25', 'Trump OR Clinton since:2015-04-25 until:2015-12-19', 'Trump OR Clinton since:2015-12-19 until:2016-08-12', 'Trump OR Clinton since:2016-08-12 until:2017-04-06', 'Trump OR Clinton since:2017-04-06 until:2017-11-30', 'Trump OR Cl

In [16]:
for tweet in query_tweets("우울증", 10):
        print(tweet.text)

INFO: queries: ['우울증 since:2006-03-21 until:2006-11-13', '우울증 since:2006-11-13 until:2007-07-08', '우울증 since:2007-07-08 until:2008-03-02', '우울증 since:2008-03-02 until:2008-10-25', '우울증 since:2008-10-25 until:2009-06-19', '우울증 since:2009-06-19 until:2010-02-12', '우울증 since:2010-02-12 until:2010-10-07', '우울증 since:2010-10-07 until:2011-06-01', '우울증 since:2011-06-01 until:2012-01-25', '우울증 since:2012-01-25 until:2012-09-18', '우울증 since:2012-09-18 until:2013-05-13', '우울증 since:2013-05-13 until:2014-01-06', '우울증 since:2014-01-06 until:2014-08-31', '우울증 since:2014-08-31 until:2015-04-25', '우울증 since:2015-04-25 until:2015-12-19', '우울증 since:2015-12-19 until:2016-08-12', '우울증 since:2016-08-12 until:2017-04-06', '우울증 since:2017-04-06 until:2017-11-30', '우울증 since:2017-11-30 until:2018-07-25', '우울증 since:2018-07-25 until:2019-03-20']
INFO: Got 0 tweets (0 new).
INFO: Got 0 tweets (0 new).
INFO: Got 20 tweets (20 new).
INFO: Got 40 tweets (20 new).
INFO: Got 60 tweets (20 new).
INFO: Got 80 tweet

DAVID FOSTER WALLACE는 우울증으로 평생 고생하고, 의사의 권고로 약을 끊었다가 병마를 이기지 못하고 2008년 자살함으로써 생을 마감. 많은 사람이 안타까워함. 그의 연설문 전문 http://bit.ly/18EnDm0 
@congjee 우울증의 초기증상일수도.
요럴땐 운동이랑 조은 사람들과 수다떨기가 특효인데.힘 내십시오.시대가 이렇더라도.
@parkkimsoojin 단순 우울.감.은 다릅니다. 우울감은 심리상담가도 다룰 수 있지만. 병적인 우울증은 정신과 상담 및 치료를 받아야 합니다. 심리상담사의 어중간한 개입으론 외려 악화되기 십상. 오픈한 유료상담소 상담 과목 한계를 아셨으면.
[네이버] 블로그 : 착한사람 증후군- 가면성 우울증과 자신을 사랑하는 법 http://me2.do/5ekdHLcN 
우울증의 증상 http://blog.naver.com/17sea/120189792792 …
당신은 자신의 비통함과 슬픔과 상처의 외로움과 갈등 심지어는 우울증까지도 밖으로 표현하는 것을 두려워할 필요가 없다. (데이빗A. 씨맨즈/상한 감정의 치유)
오늘 정신건강 테스트를 했는데 우울증이랑 약물중독을 조심하라고 했음. 별로 생활에 불만도 없고 술담배도 잘 안하는데...
고작 초등학교 저학년이던 애를 우울증에 유사자폐까지 가도록 몰아세워놓고, 겨우겨우 극복해가며 좋아진 애인데 점점 나아지니까 지들이 잘못된지도 모르는 모양이다. 철부지에, 이기적이고, 모순적이야, 진형이 부모들은. 친구로서는 몰라도 부모자격은 없어.
@201850i 뭐 그리고 우울증과 근육통, 골다공증으로 인한 힙본 골절과의 싸움도...
@ben__ur 아나 왜 이거 못봤지..ㅠㅠ;;;죄송해여 ㅠㅠ;;; 맞아여..솔까 입양보내는 저도 힘들지면 입양보내면 어미들이 우울증 많이 걸린다더라구요ㅕ ㅠㅠ;; 한마리 정도는 키워야 한다는데 그럴만큼 여유도 없고..ㅠ삐휴 ㅠ
퇴임 앞둔 교장 학교에서 목 매…우울증 심해서? | 미디어다음 http://durl.me/4ytgth 
우울증의 자석치료 ht

In [10]:
# 28개 있음

import requests
import sqlite3
import datetime
from bs4 import BeautifulSoup

# Tuple
col = ['title', 'content']
data = ()
#notice_ number
notice_num = ['11','89','8628262']

con = sqlite3.connect("data.db")
cur = con.cursor()
check = ("SELECT * FROM data")
insert = ("INSERT into data('number') VALUES (?)")
cur.execute("select * from data")


for i in range(1164,0,-1):
    req = requests.get('http://bbs.ruliweb.com/community/board/300147/list?page='+ str(i))
    html = req.text
    soup = BeautifulSoup(html, 'html.parser')

    data_numbers = soup.find_all("tr", { "class" : "table_body"})
    for m in data_numbers :
        number = m.find("td", {"class":"id"}).string.replace("\n","")
        number = number.replace(" ","")
        if(number in notice_num):
            pass
        else:
            data = data + ('('+number+')',)
            cur.execute(insert, data)
            data = ()

            
con.commit()
con.close()

In [44]:
# 저장할 내용 : 1) 제목 , 2) 내용, 3) 날짜, 4)출처 
# (0: 루리웹, 1:디씨인사이드-정신질환게시판, 2:디씨인사이드-정신병원게시판
# 3:디씨인사이드-정신과게시판 4:뽐뿌 고민상담 게시판

import requests
import sqlite3
import datetime
from bs4 import BeautifulSoup

insert = "INSERT into data(Title,Content,Date,category) VALUES (?, ?, ?, ?)"
con = sqlite3.connect("raw_data.db")
cur = con.cursor()

for i in range(0,750,1):
    req = requests.get('http://gall.dcinside.com/mgallery/board/view/?id=psychiatry&no='+str(i))
    html = req.text
    soup = BeautifulSoup(html, 'html.parser')

    data_title = soup.find_all("span", { "class" : "title_subject"})
    data_content = soup.find_all("div", { "app_paragraph" : "Dc_App_text_0"})
    if(len(data_content)==0):
        data_content = soup.find_all("div", { "style" : "overflow:hidden;"})
        for m in data_content :
            m = str(m)
            m= m.replace("\t","")
            m = m.replace("<br/>","")
            m = m.replace("</div>","")
            m = m.replace("\n","")
            m = m.replace("\r","")
            m = m.replace('<div style="overflow:hidden;">',"")
            m = m.replace('<br style="letter-spacing: -1px; background-color: rgb(255, 255, 255);"/>',"")
            m = m.replace('<p>',"")
            m = m.replace('</p>',"")
            m = m.replace('<span style="font-size: 10pt;">',"")
            m = m.replace('</span>',"")
            content = m.replace('&lt;',"")
    else:
        for m in data_content :
            m = str(m)
            m= m.replace("\t","")
            m = m.replace("<br/>","")
            m = m.replace("</div>","")
            m = m.replace("\n","")
            m = m.replace("\r","")
            content = m.replace('<div app_editorno="0" app_paragraph="Dc_App_text_0">',"")
    data_date = soup.find_all("span", { "class" : "gall_date"})
    for m in data_title :
        title = m.string.replace("\t","")
        title = title.replace("\n","")
        title = title.replace("\r","")
    for dd in data_date:
        date = dd.string
    if(len(data_content)==1):
        cur.execute(insert,(title,content,date,3))
    print(i)
con.commit()
con.close()

0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117
118
119
120
121
122
123
124
125
126
127
128
129
130
131
132
133
134
135
136
137
138
139
140
141
142
143
144
145
146
147
148
149
150
151
152
153
154
155
156
157
158
159
160
161
162
163
164
165
166
167
168
169
170
171
172
173
174
175
176
177
178
179
180
181
182
183
184
185
186
187
188
189
190
191
192
193
194
195
196
197
198
199
200
201
202
203
204
205
206
207
208
209
210
211
212
213
214
215
216
217
218
219
220
221
222
223
224
225
226
227
228
229
230
231
232
233
234
235
236
237
238
239
240
241
242
243
244
245
246
247
248
249
250
251
252
253
254
255
256
257
258
259
260
261
262
263
264
265
266
267
268
269
270
271
272
273
274
275
276
27

In [188]:
import requests
import sqlite3
import datetime
from bs4 import BeautifulSoup

con = sqlite3.connect("raw_data.db")
cur = con.cursor()

cur.execute('select * from data where Content like "%함초롬바탕%"')
tq = cur.fetchall()

con.commit()
con.close()

In [7]:
# 저장할 내용 : 1) 제목 , 2) 내용, 3) 날짜, 4)출처 
# (0: 루리웹, 1:디씨인사이드-정신질환게시판, 2:디씨인사이드-정신병원게시판
# 3:디씨인사이드-정신과게시판 4:뽐뿌 고민상담 게시판 5: 일반적인 데이터셋 - 뽐뿌 자유게시판

import requests
import sqlite3
import datetime
from bs4 import BeautifulSoup
from soynlp.normalizer import *

insert = "INSERT into data(Title,Content,Date,category) VALUES (?, ?, ?, ?)"


for i in range(5011228,5020000,1):
    con = sqlite3.connect("raw_data.db")
    cur = con.cursor()
    req = requests.get('http://www.ppomppu.co.kr/zboard/view.php?id=freeboard&no='+str(i))
    html = req.text
    soup = BeautifulSoup(html, 'html.parser')
    data_title = soup.find_all("font", { "class" : "view_title2"})
    data_content = soup.find_all("td", { "class" : "han","align":"left"})
    content = 1
    for m in data_content :
        m = str(m)
        m = only_hangle(m)
        m= m.replace('<td align="left" class="han" valign="top">',"")
        m = m.replace("<br>","")
        m = m.replace("<br/>","")
        m = m.replace("</br>","")
        m = m.replace("<div>","")
        m = m.replace("</div>","")
        m = m.replace("<p>","")
        m = m.replace("</p>","")
        m = m.replace("</td>","")
        m = m.replace("\n","")
        m = m.replace('<!--"&lt;-->',"")
        m = m.replace("함초롬바탕","")
        m = m.replace("명조체","")
        m = m.replace("궁서체","")
        m = m.replace("굴림체","")
        m = m.replace("맑은고딕","")
        m = m.replace("바탕글","")
        m = m.replace("나눔고딕","")
        m = m.replace("바탕글","")
        m = m.replace("고딕","")
        m = m.replace("돋움","")
        m = m.replace("굴림","")
        m = m.replace("궁서","")
        m = m.replace("바탕","")
        content = m

    data_date = soup.find_all("font",{"class":"eng"})
    for m in data_title :
        title = str(m)
        title = title.replace('<font class="view_title2">',"")
        title = title.replace("<!--DCM_TITLE-->","")
        title = title.replace("<!--/DCM_TITLE-->","")
        title = title.replace("</font>","")
    if(len(data_date)>0):
        date = data_date[0].text[0:10]
        temp = date
    else:
        date = temp
    if(content!=1 and len(content)>50):
        cur.execute(insert,(title,content,date,5))
        print(i)
    con.commit()
    con.close()

5011228
5011230
5011231
5011236
5011237
5011239
5011243
5011244
5011245
5011252
5011253
5011264
5011265
5011266
5011270
5011271
5011273
5011278
5011280
5011282
5011287
5011288
5011292
5011293
5011294
5011297
5011298
5011301
5011304
5011305
5011306
5011307
5011309
5011313
5011315
5011318
5011326
5011327
5011328
5011329
5011331
5011333
5011334
5011339
5011341
5011343
5011348
5011351
5011352
5011353
5011357
5011361
5011363
5011365
5011368
5011372
5011374
5011375
5011380
5011387
5011388
5011389
5011393
5011394
5011395
5011398
5011401
5011404
5011410
5011418
5011419
5011421
5011423
5011424
5011427
5011428
5011431
5011433
5011434
5011435
5011437
5011441
5011443
5011444
5011449
5011451
5011453
5011454
5011457
5011460
5011461
5011462
5011463
5011466
5011467
5011488
5011491
5011497
5011498
5011512
5011513
5011518
5011524
5011525
5011526
5011528
5011532
5011533
5011536
5011537
5011540
5011542
5011543
5011544
5011545
5011548
5011551
5011552
5011555
5011557
5011558
5011561
5011562
5011564
5011567


5013492
5013493
5013494
5013496
5013497
5013498
5013499
5013502
5013504
5013505
5013509
5013510
5013521
5013525
5013526
5013529
5013530
5013535
5013539
5013540
5013541
5013542
5013543
5013544
5013547
5013551
5013552
5013554
5013556
5013558
5013562
5013567
5013569
5013570
5013571
5013573
5013578
5013580
5013581
5013582
5013583
5013586
5013588
5013598
5013602
5013604
5013607
5013610
5013614
5013616
5013619
5013620
5013623
5013630
5013633
5013634
5013635
5013636
5013637
5013638
5013642
5013644
5013649
5013651
5013652
5013653
5013658
5013661
5013666
5013675
5013677
5013678
5013680
5013682
5013684
5013691
5013695
5013697
5013698
5013707
5013708
5013713
5013715
5013716
5013718
5013721
5013724
5013729
5013733
5013737
5013738
5013750
5013752
5013756
5013759
5013762
5013766
5013768
5013769
5013772
5013775
5013776
5013780
5013781
5013783
5013785
5013788
5013789
5013791
5013792
5013793
5013794
5013796
5013797
5013799
5013805
5013806
5013807
5013812
5013816
5013817
5013827
5013835
5013843
5013845


5016117
5016118
5016120
5016125
5016126
5016128
5016130
5016132
5016133
5016137
5016143
5016145
5016146
5016147
5016149
5016157
5016158
5016160
5016161
5016162
5016163
5016164
5016165
5016174
5016180
5016181
5016184
5016186
5016188
5016193
5016194
5016199
5016200
5016203
5016205
5016209
5016214
5016221
5016226
5016227
5016232
5016233
5016238
5016243
5016249
5016251
5016252
5016254
5016256
5016260
5016262
5016264
5016272
5016274
5016275
5016276
5016277
5016278
5016280
5016283
5016284
5016285
5016286
5016289
5016292
5016299
5016301
5016304
5016307
5016308
5016312
5016317
5016318
5016319
5016321
5016325
5016326
5016329
5016330
5016331
5016333
5016334
5016338
5016349
5016350
5016351
5016352
5016356
5016359
5016362
5016364
5016367
5016371
5016374
5016378
5016380
5016386
5016390
5016392
5016395
5016397
5016398
5016399
5016406
5016407
5016408
5016413
5016414
5016417
5016421
5016422
5016424
5016427
5016428
5016430
5016434
5016438
5016439
5016441
5016442
5016448
5016449
5016452
5016454
5016456


5018820
5018821
5018822
5018824
5018826
5018830
5018831
5018832
5018834
5018836
5018838
5018840
5018842
5018844
5018847
5018848
5018850
5018854
5018860
5018863
5018867
5018869
5018875
5018876
5018879
5018882
5018884
5018885
5018887
5018888
5018891
5018896
5018899
5018902
5018903
5018904
5018906
5018908
5018909
5018910
5018911
5018916
5018917
5018920
5018921
5018922
5018927
5018928
5018933
5018940
5018944
5018945
5018946
5018949
5018950
5018958
5018960
5018961
5018962
5018968
5018969
5018976
5018978
5018987
5018989
5018993
5018996
5018997
5019000
5019004
5019005
5019009
5019011
5019013
5019014
5019019
5019020
5019025
5019033
5019034
5019037
5019038
5019039
5019042
5019045
5019046
5019047
5019049
5019052
5019055
5019056
5019057
5019059
5019060
5019066
5019068
5019070
5019073
5019077
5019080
5019084
5019087
5019102
5019106
5019109
5019110
5019113
5019116
5019119
5019121
5019126
5019127
5019128
5019129
5019131
5019140
5019141
5019143
5019146
5019149
5019152
5019153
5019154
5019159
5019163


In [6]:
con.commit()
con.close()

In [15]:
import requests
import sqlite3
from bs4 import BeautifulSoup

insert = "INSERT into data(Title,Content,Date,category) VALUES (?, ?, ?, ?)"
con = sqlite3.connect("raw_data.db")
cur = con.cursor()

for i in range(0,len(dd),1):
    req = requests.get('http://bbs.ruliweb.com/community/board/300147/read/'+dd[i][0])
    html = req.text
    soup = BeautifulSoup(html, 'html.parser')

    data_title = soup.find_all("span", { "class" : "subject_text"})
    data_content = soup.find_all("div", { "class" : "view_content"})    
    data_date = soup.find_all("span", { "class" : "regdate"})

    for m in data_title :
        title = m.string.replace("\t","")
        title = title.replace("\n","")
    for m in data_content :
        m = str(m)
        m= m.replace("\t","")
        m = m.replace("<p>","")
        m = m.replace("</p>","")
        m = m.replace("<span style=\"font-size: 12px;\">","")
        m = m.replace("</span>","")
        m = m.replace("<div class=\"view_content\">","")
        m = m.replace("</div>","")
        m = m.replace("<p style=\"font-size: 12px;\">","")
        m = m.replace("<br>","")
        m = m.replace("<br/>","")
        m = m.replace("\n","")
        m = m.replace("\r","")
        content = m.replace("</br>","")

    for m in data_date:
        date = m.string
    cur.execute(insert,(title,content,date,0))
    print(i)
con.commit()
con.close()

0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117
118
119
120
121
122
123
124
125
126
127
128
129
130
131
132
133
134
135
136
137
138
139
140
141
142
143
144
145
146
147
148
149
150
151
152
153
154
155
156
157
158
159
160
161
162
163
164
165
166
167
168
169
170
171
172
173
174
175
176
177
178
179
180
181
182
183
184
185
186
187
188
189
190
191
192
193
194
195
196
197
198
199
200
201
202
203
204
205
206
207
208
209
210
211
212
213
214
215
216
217
218
219
220
221
222
223
224
225
226
227
228
229
230
231
232
233
234
235
236
237
238
239
240
241
242
243
244
245
246
247
248
249
250
251
252
253
254
255
256
257
258
259
260
261
262
263
264
265
266
267
268
269
270
271
272
273
274
275
276
27

1861
1862
1863
1864
1865
1866
1867
1868
1869
1870
1871
1872
1873
1874
1875
1876
1877
1878
1879
1880
1881
1882
1883
1884
1885
1886
1887
1888
1889
1890
1891
1892
1893
1894
1895
1896
1897
1898
1899
1900
1901
1902
1903
1904
1905
1906
1907
1908
1909
1910
1911
1912
1913
1914
1915
1916
1917
1918
1919
1920
1921
1922
1923
1924
1925
1926
1927
1928
1929
1930
1931
1932
1933
1934
1935
1936
1937
1938
1939
1940
1941
1942
1943
1944
1945
1946
1947
1948
1949
1950
1951
1952
1953
1954
1955
1956
1957
1958
1959
1960
1961
1962
1963
1964
1965
1966
1967
1968
1969
1970
1971
1972
1973
1974
1975
1976
1977
1978
1979
1980
1981
1982
1983
1984
1985
1986
1987
1988
1989
1990
1991
1992
1993
1994
1995
1996
1997
1998
1999
2000
2001
2002
2003
2004
2005
2006
2007
2008
2009
2010
2011
2012
2013
2014
2015
2016
2017
2018
2019
2020
2021
2022
2023
2024
2025
2026
2027
2028
2029
2030
2031
2032
2033
2034
2035
2036
2037
2038
2039
2040
2041
2042
2043
2044
2045
2046
2047
2048
2049
2050
2051
2052
2053
2054
2055
2056
2057
2058
2059
2060


3500
3501
3502
3503
3504
3505
3506
3507
3508
3509
3510
3511
3512
3513
3514
3515
3516
3517
3518
3519
3520
3521
3522
3523
3524
3525
3526
3527
3528
3529
3530
3531
3532
3533
3534
3535
3536
3537
3538
3539
3540
3541
3542
3543
3544
3545
3546
3547
3548
3549
3550
3551
3552
3553
3554
3555
3556
3557
3558
3559
3560
3561
3562
3563
3564
3565
3566
3567
3568
3569
3570
3571
3572
3573
3574
3575
3576
3577
3578
3579
3580
3581
3582
3583
3584
3585
3586
3587
3588
3589
3590
3591
3592
3593
3594
3595
3596
3597
3598
3599
3600
3601
3602
3603
3604
3605
3606
3607
3608
3609
3610
3611
3612
3613
3614
3615
3616
3617
3618
3619
3620
3621
3622
3623
3624
3625
3626
3627
3628
3629
3630
3631
3632
3633
3634
3635
3636
3637
3638
3639
3640
3641
3642
3643
3644
3645
3646
3647
3648
3649
3650
3651
3652
3653
3654
3655
3656
3657
3658
3659
3660
3661
3662
3663
3664
3665
3666
3667
3668
3669
3670
3671
3672
3673
3674
3675
3676
3677
3678
3679
3680
3681
3682
3683
3684
3685
3686
3687
3688
3689
3690
3691
3692
3693
3694
3695
3696
3697
3698
3699


5139
5140
5141
5142
5143
5144
5145
5146
5147
5148
5149
5150
5151
5152
5153
5154
5155
5156
5157
5158
5159
5160
5161
5162
5163
5164
5165
5166
5167
5168
5169
5170
5171
5172
5173
5174
5175
5176
5177
5178
5179
5180
5181
5182
5183
5184
5185
5186
5187
5188
5189
5190
5191
5192
5193
5194
5195
5196
5197
5198
5199
5200
5201
5202
5203
5204
5205
5206
5207
5208
5209
5210
5211
5212
5213
5214
5215
5216
5217
5218
5219
5220
5221
5222
5223
5224
5225
5226
5227
5228
5229
5230
5231
5232
5233
5234
5235
5236
5237
5238
5239
5240
5241
5242
5243
5244
5245
5246
5247
5248
5249
5250
5251
5252
5253
5254
5255
5256
5257
5258
5259
5260
5261
5262
5263
5264
5265
5266
5267
5268
5269
5270
5271
5272
5273
5274
5275
5276
5277
5278
5279
5280
5281
5282
5283
5284
5285
5286
5287
5288
5289
5290
5291
5292
5293
5294
5295
5296
5297
5298
5299
5300
5301
5302
5303
5304
5305
5306
5307
5308
5309
5310
5311
5312
5313
5314
5315
5316
5317
5318
5319
5320
5321
5322
5323
5324
5325
5326
5327
5328
5329
5330
5331
5332
5333
5334
5335
5336
5337
5338


6778
6779
6780
6781
6782
6783
6784
6785
6786
6787
6788
6789
6790
6791
6792
6793
6794
6795
6796
6797
6798
6799
6800
6801
6802
6803
6804
6805
6806
6807
6808
6809
6810
6811
6812
6813
6814
6815
6816
6817
6818
6819
6820
6821
6822
6823
6824
6825
6826
6827
6828
6829
6830
6831
6832
6833
6834
6835
6836
6837
6838
6839
6840
6841
6842
6843
6844
6845
6846
6847
6848
6849
6850
6851
6852
6853
6854
6855
6856
6857
6858
6859
6860
6861
6862
6863
6864
6865
6866
6867
6868
6869
6870
6871
6872
6873
6874
6875
6876
6877
6878
6879
6880
6881
6882
6883
6884
6885
6886
6887
6888
6889
6890
6891
6892
6893
6894
6895
6896
6897
6898
6899
6900
6901
6902
6903
6904
6905
6906
6907
6908
6909
6910
6911
6912
6913
6914
6915
6916
6917
6918
6919
6920
6921
6922
6923
6924
6925
6926
6927
6928
6929
6930
6931
6932
6933
6934
6935
6936
6937
6938
6939
6940
6941
6942
6943
6944
6945
6946
6947
6948
6949
6950
6951
6952
6953
6954
6955
6956
6957
6958
6959
6960
6961
6962
6963
6964
6965
6966
6967
6968
6969
6970
6971
6972
6973
6974
6975
6976
6977


8417
8418
8419
8420
8421
8422
8423
8424
8425
8426
8427
8428
8429
8430
8431
8432
8433
8434
8435
8436
8437
8438
8439
8440
8441
8442
8443
8444
8445
8446
8447
8448
8449
8450
8451
8452
8453
8454
8455
8456
8457
8458
8459
8460
8461
8462
8463
8464
8465
8466
8467
8468
8469
8470
8471
8472
8473
8474
8475
8476
8477
8478
8479
8480
8481
8482
8483
8484
8485
8486
8487
8488
8489
8490
8491
8492
8493
8494
8495
8496
8497
8498
8499
8500
8501
8502
8503
8504
8505
8506
8507
8508
8509
8510
8511
8512
8513
8514
8515
8516
8517
8518
8519
8520
8521
8522
8523
8524
8525
8526
8527
8528
8529
8530
8531
8532
8533
8534
8535
8536
8537
8538
8539
8540
8541
8542
8543
8544
8545
8546
8547
8548
8549
8550
8551
8552
8553
8554
8555
8556
8557
8558
8559
8560
8561
8562
8563
8564
8565
8566
8567
8568
8569
8570
8571
8572
8573
8574
8575
8576
8577
8578
8579
8580
8581
8582
8583
8584
8585
8586
8587
8588
8589
8590
8591
8592
8593
8594
8595
8596
8597
8598
8599
8600
8601
8602
8603
8604
8605
8606
8607
8608
8609
8610
8611
8612
8613
8614
8615
8616


10047
10048
10049
10050
10051
10052
10053
10054
10055
10056
10057
10058
10059
10060
10061
10062
10063
10064
10065
10066
10067
10068
10069
10070
10071
10072
10073
10074
10075
10076
10077
10078
10079
10080
10081
10082
10083
10084
10085
10086
10087
10088
10089
10090
10091
10092
10093
10094
10095
10096
10097
10098
10099
10100
10101
10102
10103
10104
10105
10106
10107
10108
10109
10110
10111
10112
10113
10114
10115
10116
10117
10118
10119
10120
10121
10122
10123
10124
10125
10126
10127
10128
10129
10130
10131
10132
10133
10134
10135
10136
10137
10138
10139
10140
10141
10142
10143
10144
10145
10146
10147
10148
10149
10150
10151
10152
10153
10154
10155
10156
10157
10158
10159
10160
10161
10162
10163
10164
10165
10166
10167
10168
10169
10170
10171
10172
10173
10174
10175
10176
10177
10178
10179
10180
10181
10182
10183
10184
10185
10186
10187
10188
10189
10190
10191
10192
10193
10194
10195
10196
10197
10198
10199
10200
10201
10202
10203
10204
10205
10206
10207
10208
10209
10210
10211
10212
1021

11413
11414
11415
11416
11417
11418
11419
11420
11421
11422
11423
11424
11425
11426
11427
11428
11429
11430
11431
11432
11433
11434
11435
11436
11437
11438
11439
11440
11441
11442
11443
11444
11445
11446
11447
11448
11449
11450
11451
11452
11453
11454
11455
11456
11457
11458
11459
11460
11461
11462
11463
11464
11465
11466
11467
11468
11469
11470
11471
11472
11473
11474
11475
11476
11477
11478
11479
11480
11481
11482
11483
11484
11485
11486
11487
11488
11489
11490
11491
11492
11493
11494
11495
11496
11497
11498
11499
11500
11501
11502
11503
11504
11505
11506
11507
11508
11509
11510
11511
11512
11513
11514
11515
11516
11517
11518
11519
11520
11521
11522
11523
11524
11525
11526
11527
11528
11529
11530
11531
11532
11533
11534
11535
11536
11537
11538
11539
11540
11541
11542
11543
11544
11545
11546
11547
11548
11549
11550
11551
11552
11553
11554
11555
11556
11557
11558
11559
11560
11561
11562
11563
11564
11565
11566
11567
11568
11569
11570
11571
11572
11573
11574
11575
11576
11577
11578
1157

12779
12780
12781
12782
12783
12784
12785
12786
12787
12788
12789
12790
12791
12792
12793
12794
12795
12796
12797
12798
12799
12800
12801
12802
12803
12804
12805
12806
12807
12808
12809
12810
12811
12812
12813
12814
12815
12816
12817
12818
12819
12820
12821
12822
12823
12824
12825
12826
12827
12828
12829
12830
12831
12832
12833
12834
12835
12836
12837
12838
12839
12840
12841
12842
12843
12844
12845
12846
12847
12848
12849
12850
12851
12852
12853
12854
12855
12856
12857
12858
12859
12860
12861
12862
12863
12864
12865
12866
12867
12868
12869
12870
12871
12872
12873
12874
12875
12876
12877
12878
12879
12880
12881
12882
12883
12884
12885
12886
12887
12888
12889
12890
12891
12892
12893
12894
12895
12896
12897
12898
12899
12900
12901
12902
12903
12904
12905
12906
12907
12908
12909
12910
12911
12912
12913
12914
12915
12916
12917
12918
12919
12920
12921
12922
12923
12924
12925
12926
12927
12928
12929
12930
12931
12932
12933
12934
12935
12936
12937
12938
12939
12940
12941
12942
12943
12944
1294

14145
14146
14147
14148
14149
14150
14151
14152
14153
14154
14155
14156
14157
14158
14159
14160
14161
14162
14163
14164
14165
14166
14167
14168
14169
14170
14171
14172
14173
14174
14175
14176
14177
14178
14179
14180
14181
14182
14183
14184
14185
14186
14187
14188
14189
14190
14191
14192
14193
14194
14195
14196
14197
14198
14199
14200
14201
14202
14203
14204
14205
14206
14207
14208
14209
14210
14211
14212
14213
14214
14215
14216
14217
14218
14219
14220
14221
14222
14223
14224
14225
14226
14227
14228
14229
14230
14231
14232
14233
14234
14235
14236
14237
14238
14239
14240
14241
14242
14243
14244
14245
14246
14247
14248
14249
14250
14251
14252
14253
14254
14255
14256
14257
14258
14259
14260
14261
14262
14263
14264
14265
14266
14267
14268
14269
14270
14271
14272
14273
14274
14275
14276
14277
14278
14279
14280
14281
14282
14283
14284
14285
14286
14287
14288
14289
14290
14291
14292
14293
14294
14295
14296
14297
14298
14299
14300
14301
14302
14303
14304
14305
14306
14307
14308
14309
14310
1431

15511
15512
15513
15514
15515
15516
15517
15518
15519
15520
15521
15522
15523
15524
15525
15526
15527
15528
15529
15530
15531
15532
15533
15534
15535
15536
15537
15538
15539
15540
15541
15542
15543
15544
15545
15546
15547
15548
15549
15550
15551
15552
15553
15554
15555
15556
15557
15558
15559
15560
15561
15562
15563
15564
15565
15566
15567
15568
15569
15570
15571
15572
15573
15574
15575
15576
15577
15578
15579
15580
15581
15582
15583
15584
15585
15586
15587
15588
15589
15590
15591
15592
15593
15594
15595
15596
15597
15598
15599
15600
15601
15602
15603
15604
15605
15606
15607
15608
15609
15610
15611
15612
15613
15614
15615
15616
15617
15618
15619
15620
15621
15622
15623
15624
15625
15626
15627
15628
15629
15630
15631
15632
15633
15634
15635
15636
15637
15638
15639
15640
15641
15642
15643
15644
15645
15646
15647
15648
15649
15650
15651
15652
15653
15654
15655
15656
15657
15658
15659
15660
15661
15662
15663
15664
15665
15666
15667
15668
15669
15670
15671
15672
15673
15674
15675
15676
1567

16877
16878
16879
16880
16881
16882
16883
16884
16885
16886
16887
16888
16889
16890
16891
16892
16893
16894
16895
16896
16897
16898
16899
16900
16901
16902
16903
16904
16905
16906
16907
16908
16909
16910
16911
16912
16913
16914
16915
16916
16917
16918
16919
16920
16921
16922
16923
16924
16925
16926
16927
16928
16929
16930
16931
16932
16933
16934
16935
16936
16937
16938
16939
16940
16941
16942
16943
16944
16945
16946
16947
16948
16949
16950
16951
16952
16953
16954
16955
16956
16957
16958
16959
16960
16961
16962
16963
16964
16965
16966
16967
16968
16969
16970
16971
16972
16973
16974
16975
16976
16977
16978
16979
16980
16981
16982
16983
16984
16985
16986
16987
16988
16989
16990
16991
16992
16993
16994
16995
16996
16997
16998
16999
17000
17001
17002
17003
17004
17005
17006
17007
17008
17009
17010
17011
17012
17013
17014
17015
17016
17017
17018
17019
17020
17021
17022
17023
17024
17025
17026
17027
17028
17029
17030
17031
17032
17033
17034
17035
17036
17037
17038
17039
17040
17041
17042
1704

18243
18244
18245
18246
18247
18248
18249
18250
18251
18252
18253
18254
18255
18256
18257
18258
18259
18260
18261
18262
18263
18264
18265
18266
18267
18268
18269
18270
18271
18272
18273
18274
18275
18276
18277
18278
18279
18280
18281
18282
18283
18284
18285
18286
18287
18288
18289
18290
18291
18292
18293
18294
18295
18296
18297
18298
18299
18300
18301
18302
18303
18304
18305
18306
18307
18308
18309
18310
18311
18312
18313
18314
18315
18316
18317
18318
18319
18320
18321
18322
18323
18324
18325
18326
18327
18328
18329
18330
18331
18332
18333
18334
18335
18336
18337
18338
18339
18340
18341
18342
18343
18344
18345
18346
18347
18348
18349
18350
18351
18352
18353
18354
18355
18356
18357
18358
18359
18360
18361
18362
18363
18364
18365
18366
18367
18368
18369
18370
18371
18372
18373
18374
18375
18376
18377
18378
18379
18380
18381
18382
18383
18384
18385
18386
18387
18388
18389
18390
18391
18392
18393
18394
18395
18396
18397
18398
18399
18400
18401
18402
18403
18404
18405
18406
18407
18408
1840

19609
19610
19611
19612
19613
19614
19615
19616
19617
19618
19619
19620
19621
19622
19623
19624
19625
19626
19627
19628
19629
19630
19631
19632
19633
19634
19635
19636
19637
19638
19639
19640
19641
19642
19643
19644
19645
19646
19647
19648
19649
19650
19651
19652
19653
19654
19655
19656
19657
19658
19659
19660
19661
19662
19663
19664
19665
19666
19667
19668
19669
19670
19671
19672
19673
19674
19675
19676
19677
19678
19679
19680
19681
19682
19683
19684
19685
19686
19687
19688
19689
19690
19691
19692
19693
19694
19695
19696
19697
19698
19699
19700
19701
19702
19703
19704
19705
19706
19707
19708
19709
19710
19711
19712
19713
19714
19715
19716
19717
19718
19719
19720
19721
19722
19723
19724
19725
19726
19727
19728
19729
19730
19731
19732
19733
19734
19735
19736
19737
19738
19739
19740
19741
19742
19743
19744
19745
19746
19747
19748
19749
19750
19751
19752
19753
19754
19755
19756
19757
19758
19759
19760
19761
19762
19763
19764
19765
19766
19767
19768
19769
19770
19771
19772
19773
19774
1977

20975
20976
20977
20978
20979
20980
20981
20982
20983
20984
20985
20986
20987
20988
20989
20990
20991
20992
20993
20994
20995
20996
20997
20998
20999
21000
21001
21002
21003
21004
21005
21006
21007
21008
21009
21010
21011
21012
21013
21014
21015
21016
21017
21018
21019
21020
21021
21022
21023
21024
21025
21026
21027
21028
21029
21030
21031
21032
21033
21034
21035
21036
21037
21038
21039
21040
21041
21042
21043
21044
21045
21046
21047
21048
21049
21050
21051
21052
21053
21054
21055
21056
21057
21058
21059
21060
21061
21062
21063
21064
21065
21066
21067
21068
21069
21070
21071
21072
21073
21074
21075
21076
21077
21078
21079
21080
21081
21082
21083
21084
21085
21086
21087
21088
21089
21090
21091
21092
21093
21094
21095
21096
21097
21098
21099
21100
21101
21102
21103
21104
21105
21106
21107
21108
21109
21110
21111
21112
21113
21114
21115
21116
21117
21118
21119
21120
21121
21122
21123
21124
21125
21126
21127
21128
21129
21130
21131
21132
21133
21134
21135
21136
21137
21138
21139
21140
2114

22341
22342
22343
22344
22345
22346
22347
22348
22349
22350
22351
22352
22353
22354
22355
22356
22357
22358
22359
22360
22361
22362
22363
22364
22365
22366
22367
22368
22369
22370
22371
22372
22373
22374
22375
22376
22377
22378
22379
22380
22381
22382
22383
22384
22385
22386
22387
22388
22389
22390
22391
22392
22393
22394
22395
22396
22397
22398
22399
22400
22401
22402
22403
22404
22405
22406
22407
22408
22409
22410
22411
22412
22413
22414
22415
22416
22417
22418
22419
22420
22421
22422
22423
22424
22425
22426
22427
22428
22429
22430
22431
22432
22433
22434
22435
22436
22437
22438
22439
22440
22441
22442
22443
22444
22445
22446
22447
22448
22449
22450
22451
22452
22453
22454
22455
22456
22457
22458
22459
22460
22461
22462
22463
22464
22465
22466
22467
22468
22469
22470
22471
22472
22473
22474
22475
22476
22477
22478
22479
22480
22481
22482
22483
22484
22485
22486
22487
22488
22489
22490
22491
22492
22493
22494
22495
22496
22497
22498
22499
22500
22501
22502
22503
22504
22505
22506
2250

23707
23708
23709
23710
23711
23712
23713
23714
23715
23716
23717
23718
23719
23720
23721
23722
23723
23724
23725
23726
23727
23728
23729
23730
23731
23732
23733
23734
23735
23736
23737
23738
23739
23740
23741
23742
23743
23744
23745
23746
23747
23748
23749
23750
23751
23752
23753
23754
23755
23756
23757
23758
23759
23760
23761
23762
23763
23764
23765
23766
23767
23768
23769
23770
23771
23772
23773
23774
23775
23776
23777
23778
23779
23780
23781
23782
23783
23784
23785
23786
23787
23788
23789
23790
23791
23792
23793
23794
23795
23796
23797
23798
23799
23800
23801
23802
23803
23804
23805
23806
23807
23808
23809
23810
23811
23812
23813
23814
23815
23816
23817
23818
23819
23820
23821
23822
23823
23824
23825
23826
23827
23828
23829
23830
23831
23832
23833
23834
23835
23836
23837
23838
23839
23840
23841
23842
23843
23844
23845
23846
23847
23848
23849
23850
23851
23852
23853
23854
23855
23856
23857
23858
23859
23860
23861
23862
23863
23864
23865
23866
23867
23868
23869
23870
23871
23872
2387

25073
25074
25075
25076
25077
25078
25079
25080
25081
25082
25083
25084
25085
25086
25087
25088
25089
25090
25091
25092
25093
25094
25095
25096
25097
25098
25099
25100
25101
25102
25103
25104
25105
25106
25107
25108
25109
25110
25111
25112
25113
25114
25115
25116
25117
25118
25119
25120
25121
25122
25123
25124
25125
25126
25127
25128
25129
25130
25131
25132
25133
25134
25135
25136
25137
25138
25139
25140
25141
25142
25143
25144
25145
25146
25147
25148
25149
25150
25151
25152
25153
25154
25155
25156
25157
25158
25159
25160
25161
25162
25163
25164
25165
25166
25167
25168
25169
25170
25171
25172
25173
25174
25175
25176
25177
25178
25179
25180
25181
25182
25183
25184
25185
25186
25187
25188
25189
25190
25191
25192
25193
25194
25195
25196
25197
25198
25199
25200
25201
25202
25203
25204
25205
25206
25207
25208
25209
25210
25211
25212
25213
25214
25215
25216
25217
25218
25219
25220
25221
25222
25223
25224
25225
25226
25227
25228
25229
25230
25231
25232
25233
25234
25235
25236
25237
25238
2523

26439
26440
26441
26442
26443
26444
26445
26446
26447
26448
26449
26450
26451
26452
26453
26454
26455
26456
26457
26458
26459
26460
26461
26462
26463
26464
26465
26466
26467
26468
26469
26470
26471
26472
26473
26474
26475
26476
26477
26478
26479
26480
26481
26482
26483
26484
26485
26486
26487
26488
26489
26490
26491
26492
26493
26494
26495
26496
26497
26498
26499
26500
26501
26502
26503
26504
26505
26506
26507
26508
26509
26510
26511
26512
26513
26514
26515
26516
26517
26518
26519
26520
26521
26522
26523
26524
26525
26526
26527
26528
26529
26530
26531
26532
26533
26534
26535
26536
26537
26538
26539
26540
26541
26542
26543
26544
26545
26546
26547
26548
26549
26550
26551
26552
26553
26554
26555
26556
26557
26558
26559
26560
26561
26562
26563
26564
26565
26566
26567
26568
26569
26570
26571
26572
26573
26574
26575
26576
26577
26578
26579
26580
26581
26582
26583
26584
26585
26586
26587
26588
26589
26590
26591
26592
26593
26594
26595
26596
26597
26598
26599
26600
26601
26602
26603
26604
2660

27805
27806
27807
27808
27809
27810
27811
27812
27813
27814
27815
27816
27817
27818
27819
27820
27821
27822
27823
27824
27825
27826
27827
27828
27829
27830
27831
27832
27833
27834
27835
27836
27837
27838
27839
27840
27841
27842
27843
27844
27845
27846
27847
27848
27849
27850
27851
27852
27853
27854
27855
27856
27857
27858
27859
27860
27861
27862
27863
27864
27865
27866
27867
27868
27869
27870
27871
27872
27873
27874
27875
27876
27877
27878
27879
27880
27881
27882
27883
27884
27885
27886
27887
27888
27889
27890
27891
27892
27893
27894
27895
27896
27897
27898
27899
27900
27901
27902
27903
27904
27905
27906
27907
27908
27909
27910
27911
27912
27913
27914
27915
27916
27917
27918
27919
27920
27921
27922
27923
27924
27925
27926
27927
27928
27929
27930
27931
27932
27933
27934
27935
27936
27937
27938
27939
27940
27941
27942
27943
27944
27945
27946
27947
27948
27949
27950
27951
27952
27953
27954
27955
27956
27957
27958
27959
27960
27961
27962
27963
27964
27965
27966
27967
27968
27969
27970
2797

29171
29172
29173
29174
29175
29176
29177
29178
29179
29180
29181
29182
29183
29184
29185
29186
29187
29188
29189
29190
29191
29192
29193
29194
29195
29196
29197
29198
29199
29200
29201
29202
29203
29204
29205
29206
29207
29208
29209
29210
29211
29212
29213
29214
29215
29216
29217
29218
29219
29220
29221
29222
29223
29224
29225
29226
29227
29228
29229
29230
29231
29232
29233
29234
29235
29236
29237
29238
29239
29240
29241
29242
29243
29244
29245
29246
29247
29248
29249
29250
29251
29252
29253
29254
29255
29256
29257
29258
29259
29260
29261
29262
29263
29264
29265
29266
29267
29268
29269
29270
29271
29272
29273
29274
29275
29276
29277
29278
29279
29280
29281
29282
29283
29284
29285
29286
29287
29288
29289
29290
29291
29292
29293
29294
29295
29296
29297
29298
29299
29300
29301
29302
29303
29304
29305
29306
29307
29308
29309
29310
29311
29312
29313
29314
29315
29316
29317
29318
29319
29320
29321
29322
29323
29324
29325
29326
29327
29328
29329
29330
29331
29332
29333
29334
29335
29336
2933

30537
30538
30539
30540
30541
30542
30543
30544
30545
30546
30547
30548
30549
30550
30551
30552
30553
30554
30555
30556
30557
30558
30559
30560
30561
30562
30563
30564
30565
30566
30567
30568
30569
30570
30571
30572
30573
30574
30575
30576
30577
30578
30579
30580
30581
30582
30583
30584
30585
30586
30587
30588
30589
30590
30591
30592
30593
30594
30595
30596
30597
30598
30599
30600
30601
30602
30603
30604
30605
30606
30607
30608
30609
30610
30611
30612
30613
30614
30615
30616
30617
30618
30619
30620
30621
30622
30623
30624
30625
30626
30627
30628
30629
30630
30631
30632
30633
30634
30635
30636
30637
30638
30639
30640
30641
30642
30643
30644
30645
30646
30647
30648
30649
30650
30651
30652
30653
30654
30655
30656
30657
30658
30659
30660
30661
30662
30663
30664
30665
30666
30667
30668
30669
30670
30671
30672
30673
30674
30675
30676
30677
30678
30679
30680
30681
30682
30683
30684
30685
30686
30687
30688
30689
30690
30691
30692
30693
30694
30695
30696
30697
30698
30699
30700
30701
30702
3070

31903
31904
31905
31906
31907
31908
31909
31910
31911
31912
31913
31914
31915
31916
31917
31918
31919
31920
31921
31922
31923
31924
31925
31926
31927
31928
31929
31930
31931
31932
31933
31934
31935
31936
31937
31938
31939
31940
31941
31942
31943
31944
31945
31946
31947
31948
31949
31950
31951
31952
31953
31954
31955
31956
31957
31958
31959
31960
31961
31962
31963
31964
31965
31966
31967
31968
31969
31970
31971
31972
31973
31974
31975
31976
31977
31978
31979
31980
31981
31982
31983
31984
31985
31986
31987
31988
31989
31990
31991
31992
31993
31994
31995
31996
31997
31998
31999
32000
32001
32002
32003
32004
32005
32006
32007
32008
32009
32010
32011
32012
32013
32014
32015
32016
32017
32018
32019
32020
32021
32022
32023
32024
32025
32026
32027
32028
32029
32030
32031
32032
32033
32034
32035
32036
32037
32038
32039
32040
32041
32042
32043
32044
32045
32046
32047
32048
32049
32050
32051
32052
32053
32054
32055
32056
32057
32058
32059
32060
32061
32062
32063
32064
32065
32066
32067
32068
3206

In [1]:
# -*- coding: utf-8 -*- 

from selenium import webdriver
from bs4 import BeautifulSoup as bs
import pandas as pd
import time
import random

driver = webdriver.Chrome('./chromedriver.exe')
driver.implicitly_wait(3)
# 네이버 메인에 최초 한 번 접근 후에 로그인 페이지로 접속하는 방법이다. (다른 분의 글에서 도움을 받았다)
driver.get("https://www.daum.net/") 
time.sleep(random.randrange(2,5))
driver.find_element_by_css_selector('#btnKakaoLogin').click() # 로그인 페이지 접속
time.sleep(random.randrange(3,6))
# 아이디와 비밀번호도 키보드가 입력한 것처럼 하나하나씩 입력한다.
# for id in 'kch21026':
#     time.sleep(0.2)
#     driver.find_element_by_name('id').send_keys(id)
# time.sleep(random.randrange(1,3))
# for pw in 'eksldpf4106*':
#     time.sleep(0.2)
#     driver.find_element_by_name('pw').send_keys(pw)
# time.sleep(random.randrange(1,3)) 
# driver.find_element_by_css_selector('#frmNIDLogin > fieldset > input').click()

NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#btnKakaoLogin"}
  (Session info: chrome=74.0.3729.108)
  (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.18342 x86_64)


In [2]:
base_url = 'https://cafe.naver.com/koreanmania/ArticleList.nhn?search.clubid=11276433'
base_url = 'http://cafe.daum.net/overcomemyself/GOqz/'

In [38]:
# driver.get(base_url + '&search.menuid=17&search.page='+i)
driver.get(base_url + '132')
driver.switch_to.default_content()
driver.switch_to.frame("down")
body = driver.find_element_by_css_selector("html > body")
try:
    post_subject = body.find_element_by_xpath("//div[@class='article_subject line_sub']")
    post_info = body.find_element_by_xpath("//div[@class='article_writer']")
    post_contents = body.find_element_by_xpath("//div[@class='bbs_contents']")
    subject = post_subject.find_element_by_xpath("//span[@class='b']").text
    date_time = post_info.find_element_by_xpath("//span[@class='p11 ls0']").text[:10]
    contents = post_contents.find_element_by_xpath("//table[@class='protectTable']").text
    print(subject)
    print(date_time)
except Exception as e:
    None


# iframe으로 프레임 전환
# driver.switch_to_frame('cafe_main')

type: 1 조현병(정신분열증) - 자유게시판
      2 조현병(정신분열증) - 상담실
      3 ADHD - 자유게시판
      4 ADHD - 익명게시판

In [9]:
con.commit()
con.close()

In [17]:
print("aa")

aa


In [5]:
# -*- coding: utf-8 -*- 

from selenium import webdriver
from bs4 import BeautifulSoup as bs
import pandas as pd
import time
import random
import sqlite3
import requests


insert = "INSERT into data(id,title,content,date,type) VALUES (?, ?, ?, ?, ?)"
base_url = 'http://cafe.daum.net/Attentiondeficit/8mW1/'
#5524
r = 1354
for i in range(2206,5524):
    con = sqlite3.connect("cafe.db")
    cur = con.cursor()
    driver.get(base_url + str(i))
    driver.switch_to.default_content()
    driver.switch_to.frame("down")
    body = driver.find_element_by_css_selector("html > body")
    res_list = []
    try:
        post_subject = body.find_element_by_xpath("//div[@class='article_subject line_sub']")
        post_info = body.find_element_by_xpath("//div[@class='article_writer']")
        post_contents = body.find_element_by_xpath("//div[@class='bbs_contents']")
        title = post_subject.find_element_by_xpath("//span[@class='b']").text
        date_time = post_info.find_element_by_xpath("//span[@class='p11 ls0']").text[:10]
        content = post_contents.find_element_by_xpath("//table[@class='protectTable']").text
        cafe_df = pd.DataFrame(res_list)
        cur.execute(insert,(r,title,content,date_time,4))
        print(i)
        r = r + 1
        con.commit()
        con.close()
    except Exception as e:
        None

2207
2208
2209
2215
2217
2219
2220
2221
2222
2223
2224
2225
2226
2227
2228
2229
2230
2231
2232
2234
2235
2236
2237
2238
2239
2240
2241
2242
2244
2245
2246
2247
2248
2249
2250
2251
2252
2254
2255
2256
2257
2258
2259
2260
2261
2262
2263
2264
2265
2266
2267
2268
2269
2270
2271
2272
2273
2274
2276
2277
2278
2279
2280
2281
2282
2283
2284
2285
2286
2287
2288
2289
2290
2291
2292
2294
2295
2296
2297
2298
2299
2300
2301
2302
2303
2304
2305
2306
2309
2310
2311
2312
2313
2314
2315
2316
2317
2318
2320
2322
2324
2325
2327
2328
2329
2330
2331
2332
2334
2335
2336
2337
2339
2340
2341
2342
2343
2345
2346
2347
2348
2349
2350
2351
2352
2353
2355
2356
2357
2358
2359
2360
2361
2362
2363
2364
2365
2367
2369
2370
2372
2373
2374
2376
2377
2378
2379
2380
2381
2382
2383
2384
2385
2387
2389
2390
2391
2392
2393
2395
2396
2397
2398
2399
2400
2401
2402
2403
2404
2405
2406
2407
2408
2409
2410
2412
2413
2416
2417
2418
2419
2421
2422
2423
2424
2425
2427
2429
2430
2431
2433
2434
2436
2437
2438
2439
2440
2441
2442
2443


4452
4453
4454
4455
4456
4457
4458
4461
4463
4464
4465
4466
4467
4468
4469
4473
4474
4475
4477
4481
4482
4484
4485
4487
4489
4490
4491
4493
4494
4499
4500
4504
4506
4507
4508
4509
4513
4514
4515
4516
4517
4518
4519
4520
4521
4523
4526
4527
4529
4530
4534
4537
4538
4539
4542
4543
4544
4545
4546
4547
4548
4550
4551
4552
4553
4554
4555
4558
4559
4560
4561
4562
4568
4570
4571
4572
4573
4574
4575
4576
4580
4581
4582
4583
4584
4585
4586
4587
4588
4589
4590
4591
4593
4594
4595
4596
4597
4598
4599
4600
4601
4603
4604
4605
4606
4607
4608
4611
4613
4614
4617
4618
4619
4621
4623
4624
4627
4633
4634
4636
4637
4640
4643
4645
4646
4648
4650
4653
4654
4655
4656
4657
4658
4659
4660
4661
4663
4664
4665
4666
4667
4670
4673
4674
4675
4676
4677
4678
4680
4681
4683
4685
4686
4687
4688
4690
4693
4695
4696
4697
4698
4699
4700
4701
4702
4704
4705
4706
4708
4709
4710
4711
4712
4713
4714
4715
4716
4717
4718
4721
4722
4723
4724
4725
4727
4728
4729
4731
4732
4733
4735
4737
4738
4741
4742
4743
4745
4746
4748
4750


In [11]:
for i in range
driver.get(base_url + '&search.menuid=24&search.page='+i)
# iframe으로 프레임 전환
driver.switch_to_frame('cafe_main')
# href 속성을 찾아 url을 리스트로 저장한다.
article_list = driver.find_elements_by_css_selector('span.aaa > a.m-tcol-c')
article_urls = [ i.get_attribute('href') for i in article_list ]
res_list = []
# Beautifulsoup 활용
for article in article_urls:
    driver.get(article)
    # article도 switch_to_frame이 필수
    driver.switch_to_frame('cafe_main')
    soup = bs(driver.page_source, 'html.parser')
    # 게시글에서 제목 추출
    title = soup.select('div.tit-box span.b')[0].get_text()
    # 내용을 하나의 텍스트로 만든다. (띄어쓰기 단위)
    content_tags = soup.select('#tbody')[0].select('p')
    content = ' '.join([ tags.get_text() for tags in content_tags ])
    # dict형태로 만들어 결과 list에 저장
    res_list.append({'title' : title, 'content' : content})
    # time.sleep 작업도 필요하다.
# 결과 데이터프레임화
cafe_df = pd.DataFrame(res_list)
# csv파일로 추출
cafe_df.to_csv('cafe_crawling.csv', index=False)

C:\anaconda3\lib\site-packages\ipykernel_launcher.py:6: DeprecationWarning: use driver.switch_to.frame instead
  


In [113]:
k = 1

In [116]:
k

176

In [115]:
import sqlite3
from bs4 import BeautifulSoup

insert = "INSERT into data(id,title,content,category) VALUES (?, ?, ?, ?)"
base_url = 'https://cafe.naver.com/koreanmania/ArticleList.nhn?search.clubid=11276433'
for i in range(1,13):
    con = sqlite3.connect("cafe_data.db")
    cur = con.cursor()
    driver.get(base_url + '&search.menuid=12&search.page='+str(i))
    # iframe으로 프레임 전환
    driver.switch_to_frame('cafe_main')
    # href 속성을 찾아 url을 리스트로 저장한다.
    article_list = driver.find_elements_by_css_selector('span.aaa > a.m-tcol-c')
    article_urls = [ i.get_attribute('href') for i in article_list ]
    res_list = []
    # Beautifulsoup 활용
    for article in article_urls:
        driver.get(article)
        # article도 switch_to_frame이 필수
        driver.switch_to_frame('cafe_main')
        soup = bs(driver.page_source, 'html.parser')
        # 게시글에서 제목 추출
        title = soup.select('div.tit-box span.b')[0].get_text()
        # 내용을 하나의 텍스트로 만든다. (띄어쓰기 단위)
        content_tags = soup.select('#tbody')[0].select('p')
        content = ' '.join([ tags.get_text() for tags in content_tags ])
        content = content.replace("\xa0","")
        cur.execute(insert,(k,title,content,9))
        k += 1
        # time.sleep 작업도 필요하다.
    con.commit()
    con.close()

C:\anaconda3\lib\site-packages\ipykernel_launcher.py:11: DeprecationWarning: use driver.switch_to.frame instead
  # This is added back by InteractiveShellApp.init_path()
C:\anaconda3\lib\site-packages\ipykernel_launcher.py:20: DeprecationWarning: use driver.switch_to.frame instead


table 구조<br>
id, title, content, category<br>
category : 0.조증 / 1.우울증 / 2.조울증 / 3.과잉행동장애 / 4. 정신분열증 /<br>
<t>5.강박증/ 6.불면증 / 7.공황장애 / 8.노이로제(신경증) / 9.외상후 스트레스 장애

조증 : menu_id:17 / 29 page<br>
우울증 : menu_id:24 / 659 page<br>
조울증 : menu_id:8 / 766 page<br>
과잉행동장애 : menu_id:133 / 2 page<br>
정신분열증(조현증) : menu_id:11 / 266 page<br>
강박증 : menu_id:194 / 10 page<br>
불면증 : menu_id:231 / 36 page<br>
공황장애 : menu_id:240 / 14 page<br>
노이로제(신경증) : menu_id:122 / 12 page<br>
외상후 스트레스 : menu_id:12 / 12 page<br>
한 page 에 15개의 게시글

In [3]:
# 저장할 내용 : 1) 제목 , 2) 내용, 3) 날짜
# dc인사이드 아주대학교 갤러리

import requests
import sqlite3
import datetime
from bs4 import BeautifulSoup
from soynlp.normalizer import *

insert = "INSERT into data(title,content,date) VALUES (?, ?, ?)"

#627793
for i in range(610266,627793,1):
    con = sqlite3.connect("ajou_data.db")
    cur = con.cursor()
    req = requests.get('http://gall.dcinside.com/board/view/?id=ajou&no='+str(i))
    html = req.text
    soup = BeautifulSoup(html, 'html.parser')

    data_title = soup.find_all("span", { "class" : "title_subject"})
    data_content = soup.find_all("div", { "app_paragraph" : "Dc_App_text_0"})
    if(len(data_content)==0):
        data_content = soup.find_all("div", { "style" : "overflow:hidden;"})
        for m in data_content :
            m = str(m)
            m= m.replace("\t","")
            m = m.replace("<br/>","")
            m = m.replace("</div>","")
            m = m.replace("\n","")
            m = m.replace("\r","")
            m = m.replace('<div style="overflow:hidden;">',"")
            m = m.replace('<br style="letter-spacing: -1px; background-color: rgb(255, 255, 255);"/>',"")
            m = m.replace('<p>',"")
            m = m.replace('</p>',"")
            m = m.replace('<span style="font-size: 10pt;">',"")
            m = m.replace('</span>',"")
            content = m.replace('&lt;',"")
    else:
        for m in data_content :
            m = str(m)
            m= m.replace("\t","")
            m = m.replace("<br/>","")
            m = m.replace("</div>","")
            m = m.replace("\n","")
            m = m.replace("\r","")
            content = m.replace('<div app_editorno="0" app_paragraph="Dc_App_text_0">',"")
    content = only_hangle(content)
    data_date = soup.find_all("span", { "class" : "gall_date"})
    for m in data_title :
        title = m.string.replace("\t","")
        title = title.replace("\n","")
        title = title.replace("\r","")
    for dd in data_date:
        date = dd.string
    if(len(data_content)==1):
        cur.execute(insert,(title,content,date))
    print(i)
    con.commit()
    con.close()

610266
610267
610268
610269
610270
610271
610272
610273
610274
610275
610276
610277
610278
610279
610280
610281
610282
610283
610284
610285
610286
610287
610288
610289
610290
610291
610292
610293
610294
610295
610296
610297
610298
610299
610300
610301
610302
610303
610304
610305
610306
610307
610308
610309
610310
610311
610312
610313
610314
610315
610316
610317
610318
610319
610320
610321
610322
610323
610324
610325
610326
610327
610328
610329
610330
610331
610332
610333
610334
610335
610336
610337
610338
610339
610340
610341
610342
610343
610344
610345
610346
610347
610348
610349
610350
610351
610352
610353
610354
610355
610356
610357
610358
610359
610360
610361
610362
610363
610364
610365
610366
610367
610368
610369
610370
610371
610372
610373
610374
610375
610376
610377
610378
610379
610380
610381
610382
610383
610384
610385
610386
610387
610388
610389
610390
610391
610392
610393
610394
610395
610396
610397
610398
610399
610400
610401
610402
610403
610404
610405
610406
610407
610408

611437
611438
611439
611440
611441
611442
611443
611444
611445
611446
611447
611448
611449
611450
611451
611452
611453
611454
611455
611456
611457
611458
611459
611460
611461
611462
611463
611464
611465
611466
611467
611468
611469
611470
611471
611472
611473
611474
611475
611476
611477
611478
611479
611480
611481
611482
611483
611484
611485
611486
611487
611488
611489
611490
611491
611492
611493
611494
611495
611496
611497
611498
611499
611500
611501
611502
611503
611504
611505
611506
611507
611508
611509
611510
611511
611512
611513
611514
611515
611516
611517
611518
611519
611520
611521
611522
611523
611524
611525
611526
611527
611528
611529
611530
611531
611532
611533
611534
611535
611536
611537
611538
611539
611540
611541
611542
611543
611544
611545
611546
611547
611548
611549
611550
611551
611552
611553
611554
611555
611556
611557
611558
611559
611560
611561
611562
611563
611564
611565
611566
611567
611568
611569
611570
611571
611572
611573
611574
611575
611576
611577
611578
611579

612608
612609
612610
612611
612612
612613
612614
612615
612616
612617
612618
612619
612620
612621
612622
612623
612624
612625
612626
612627
612628
612629
612630
612631
612632
612633
612634
612635
612636
612637
612638
612639
612640
612641
612642
612643
612644
612645
612646
612647
612648
612649
612650
612651
612652
612653
612654
612655
612656
612657
612658
612659
612660
612661
612662
612663
612664
612665
612666
612667
612668
612669
612670
612671
612672
612673
612674
612675
612676
612677
612678
612679
612680
612681
612682
612683
612684
612685
612686
612687
612688
612689
612690
612691
612692
612693
612694
612695
612696
612697
612698
612699
612700
612701
612702
612703
612704
612705
612706
612707
612708
612709
612710
612711
612712
612713
612714
612715
612716
612717
612718
612719
612720
612721
612722
612723
612724
612725
612726
612727
612728
612729
612730
612731
612732
612733
612734
612735
612736
612737
612738
612739
612740
612741
612742
612743
612744
612745
612746
612747
612748
612749
612750

613782
613783
613784
613785
613786
613787
613788
613789
613790
613791
613792
613793
613794
613795
613796
613797
613798
613799
613800
613801
613802
613803
613804
613805
613806
613807
613808
613809
613810
613811
613812
613813
613814
613815
613816
613817
613818
613819
613820
613821
613822
613823
613824
613825
613826
613827
613828
613829
613830
613831
613832
613833
613834
613835
613836
613837
613838
613839
613840
613841
613842
613843
613844
613845
613846
613847
613848
613849
613850
613851
613852
613853
613854
613855
613856
613857
613858
613859
613860
613861
613862
613863
613864
613865
613866
613867
613868
613869
613870
613871
613872
613873
613874
613875
613876
613877
613878
613879
613880
613881
613882
613883
613884
613885
613886
613887
613888
613889
613890
613891
613892
613893
613894
613895
613896
613897
613898
613899
613900
613901
613902
613903
613904
613905
613906
613907
613908
613909
613910
613911
613912
613913
613914
613915
613916
613917
613918
613919
613920
613921
613922
613923
613924

614953
614954
614955
614956
614957
614958
614959
614960
614961
614962
614963
614964
614965
614966
614967
614968
614969
614970
614971
614972
614973
614974
614975
614976
614977
614978
614979
614980
614981
614982
614983
614984
614985
614986
614987
614988
614989
614990
614991
614992
614993
614994
614995
614996
614997
614998
614999
615000
615001
615002
615003
615004
615005
615006
615007
615008
615009
615010
615011
615012
615013
615014
615015
615016
615017
615018
615019
615020
615021
615022
615023
615024
615025
615026
615027
615028
615029
615030
615031
615032
615033
615034
615035
615036
615037
615038
615039
615040
615041
615042
615043
615044
615045
615046
615047
615048
615049
615050
615051
615052
615053
615054
615055
615056
615057
615058
615059
615060
615061
615062
615063
615064
615065
615066
615067
615068
615069
615070
615071
615072
615073
615074
615075
615076
615077
615078
615079
615080
615081
615082
615083
615084
615085
615086
615087
615088
615089
615090
615091
615092
615093
615094
615095

616124
616125
616126
616127
616128
616129
616130
616131
616132
616133
616134
616135
616136
616137
616138
616139
616140
616141
616142
616143
616144
616145
616146
616147
616148
616149
616150
616151
616152
616153
616154
616155
616156
616157
616158
616159
616160
616161
616162
616163
616164
616165
616166
616167
616168
616169
616170
616171
616172
616173
616174
616175
616176
616177
616178
616179
616180
616181
616182
616183
616184
616185
616186
616187
616188
616189
616190
616191
616192
616193
616194
616195
616196
616197
616198
616199
616200
616201
616202
616203
616204
616205
616206
616207
616208
616209
616210
616211
616212
616213
616214
616215
616216
616217
616218
616219
616220
616221
616222
616223
616224
616225
616226
616227
616228
616229
616230
616231
616232
616233
616234
616235
616236
616237
616238
616239
616240
616241
616242
616243
616244
616245
616246
616247
616248
616249
616250
616251
616252
616253
616254
616255
616256
616257
616258
616259
616260
616261
616262
616263
616264
616265
616266

617295
617296
617297
617298
617299
617300
617301
617302
617303
617304
617305
617306
617307
617308
617309
617310
617311
617312
617313
617314
617315
617316
617317
617318
617319
617320
617321
617322
617323
617324
617325
617326
617327
617328
617329
617330
617331
617332
617333
617334
617335
617336
617337
617338
617339
617340
617341
617342
617343
617344
617345
617346
617347
617348
617349
617350
617351
617352
617353
617354
617355
617356
617357
617358
617359
617360
617361
617362
617363
617364
617365
617366
617367
617368
617369
617370
617371
617372
617373
617374
617375
617376
617377
617378
617379
617380
617381
617382
617383
617384
617385
617386
617387
617388
617389
617390
617391
617392
617393
617394
617395
617396
617397
617398
617399
617400
617401
617402
617403
617404
617405
617406
617407
617408
617409
617410
617411
617412
617413
617414
617415
617416
617417
617418
617419
617420
617421
617422
617423
617424
617425
617426
617427
617428
617429
617430
617431
617432
617433
617434
617435
617436
617437

618466
618467
618468
618469
618470
618471
618472
618473
618474
618475
618476
618477
618478
618479
618480
618481
618482
618483
618484
618485
618486
618487
618488
618489
618490
618491
618492
618493
618494
618495
618496
618497
618498
618499
618500
618501
618502
618503
618504
618505
618506
618507
618508
618509
618510
618511
618512
618513
618514
618515
618516
618517
618518
618519
618520
618521
618522
618523
618524
618525
618526
618527
618528
618529
618530
618531
618532
618533
618534
618535
618536
618537
618538
618539
618540
618541
618542
618543
618544
618545
618546
618547
618548
618549
618550
618551
618552
618553
618554
618555
618556
618557
618558
618559
618560
618561
618562
618563
618564
618565
618566
618567
618568
618569
618570
618571
618572
618573
618574
618575
618576
618577
618578
618579
618580
618581
618582
618583
618584
618585
618586
618587
618588
618589
618590
618591
618592
618593
618594
618595
618596
618597
618598
618599
618600
618601
618602
618603
618604
618605
618606
618607
618608

619637
619638
619639
619640
619641
619642
619643
619644
619645
619646
619647
619648
619649
619650
619651
619652
619653
619654
619655
619656
619657
619658
619659
619660
619661
619662
619663
619664
619665
619666
619667
619668
619669
619670
619671
619672
619673
619674
619675
619676
619677
619678
619679
619680
619681
619682
619683
619684
619685
619686
619687
619688
619689
619690
619691
619692
619693
619694
619695
619696
619697
619698
619699
619700
619701
619702
619703
619704
619705
619706
619707
619708
619709
619710
619711
619712
619713
619714
619715
619716
619717
619718
619719
619720
619721
619722
619723
619724
619725
619726
619727
619728
619729
619730
619731
619732
619733
619734
619735
619736
619737
619738
619739
619740
619741
619742
619743
619744
619745
619746
619747
619748
619749
619750
619751
619752
619753
619754
619755
619756
619757
619758
619759
619760
619761
619762
619763
619764
619765
619766
619767
619768
619769
619770
619771
619772
619773
619774
619775
619776
619777
619778
619779

620808
620809
620810
620811
620812
620813
620814
620815
620816
620817
620818
620819
620820
620821
620822
620823
620824
620825
620826
620827
620828
620829
620830
620831
620832
620833
620834
620835
620836
620837
620838
620839
620840
620841
620842
620843
620844
620845
620846
620847
620848
620849
620850
620851
620852
620853
620854
620855
620856
620857
620858
620859
620860
620861
620862
620863
620864
620865
620866
620867
620868
620869
620870
620871
620872
620873
620874
620875
620876
620877
620878
620879
620880
620881
620882
620883
620884
620885
620886
620887
620888
620889
620890
620891
620892
620893
620894
620895
620896
620897
620898
620899
620900
620901
620902
620903
620904
620905
620906
620907
620908
620909
620910
620911
620912
620913
620914
620915
620916
620917
620918
620919
620920
620921
620922
620923
620924
620925
620926
620927
620928
620929
620930
620931
620932
620933
620934
620935
620936
620937
620938
620939
620940
620941
620942
620943
620944
620945
620946
620947
620948
620949
620950

621979
621980
621981
621982
621983
621984
621985
621986
621987
621988
621989
621990
621991
621992
621993
621994
621995
621996
621997
621998
621999
622000
622001
622002
622003
622004
622005
622006
622007
622008
622009
622010
622011
622012
622013
622014
622015
622016
622017
622018
622019
622020
622021
622022
622023
622024
622025
622026
622027
622028
622029
622030
622031
622032
622033
622034
622035
622036
622037
622038
622039
622040
622041
622042
622043
622044
622045
622046
622047
622048
622049
622050
622051
622052
622053
622054
622055
622056
622057
622058
622059
622060
622061
622062
622063
622064
622065
622066
622067
622068
622069
622070
622071
622072
622073
622074
622075
622076
622077
622078
622079
622080
622081
622082
622083
622084
622085
622086
622087
622088
622089
622090
622091
622092
622093
622094
622095
622096
622097
622098
622099
622100
622101
622102
622103
622104
622105
622106
622107
622108
622109
622110
622111
622112
622113
622114
622115
622116
622117
622118
622119
622120
622121

623150
623151
623152
623153
623154
623155
623156
623157
623158
623159
623160
623161
623162
623163
623164
623165
623166
623167
623168
623169
623170
623171
623172
623173
623174
623175
623176
623177
623178
623179
623180
623181
623182
623183
623184
623185
623186
623187
623188
623189
623190
623191
623192
623193
623194
623195
623196
623197
623198
623199
623200
623201
623202
623203
623204
623205
623206
623207
623208
623209
623210
623211
623212
623213
623214
623215
623216
623217
623218
623219
623220
623221
623222
623223
623224
623225
623226
623227
623228
623229
623230
623231
623232
623233
623234
623235
623236
623237
623238
623239
623240
623241
623242
623243
623244
623245
623246
623247
623248
623249
623250
623251
623252
623253
623254
623255
623256
623257
623258
623259
623260
623261
623262
623263
623264
623265
623266
623267
623268
623269
623270
623271
623272
623273
623274
623275
623276
623277
623278
623279
623280
623281
623282
623283
623284
623285
623286
623287
623288
623289
623290
623291
623292

624321
624322
624323
624324
624325
624326
624327
624328
624329
624330
624331
624332
624333
624334
624335
624336
624337
624338
624339
624340
624341
624342
624343
624344
624345
624346
624347
624348
624349
624350
624351
624352
624353
624354
624355
624356
624357
624358
624359
624360
624361
624362
624363
624364
624365
624366
624367
624368
624369
624370
624371
624372
624373
624374
624375
624376
624377
624378
624379
624380
624381
624382
624383
624384
624385
624386
624387
624388
624389
624390
624391
624392
624393
624394
624395
624396
624397
624398
624399
624400
624401
624402
624403
624404
624405
624406
624407
624408
624409
624410
624411
624412
624413
624414
624415
624416
624417
624418
624419
624420
624421
624422
624423
624424
624425
624426
624427
624428
624429
624430
624431
624432
624433
624434
624435
624436
624437
624438
624439
624440
624441
624442
624443
624444
624445
624446
624447
624448
624449
624450
624451
624452
624453
624454
624455
624456
624457
624458
624459
624460
624461
624462
624463

625493
625494
625495
625496
625497
625498
625499
625500
625501
625502
625503
625504
625505
625506
625507
625508
625509
625510
625511
625512
625513
625514
625515
625516
625517
625518
625519
625520
625521
625522
625523
625524
625525
625526
625527
625528
625529
625530
625531
625532
625533
625534
625535
625536
625537
625538
625539
625540
625541
625542
625543
625544
625545
625546
625547
625548
625549
625550
625551
625552
625553
625554
625555
625556
625557
625558
625559
625560
625561
625562
625563
625564
625565
625566
625567
625568
625569
625570
625571
625572
625573
625574
625575
625576
625577
625578
625579
625580
625581
625582
625583
625584
625585
625586
625587
625588
625589
625590
625591
625592
625593
625594
625595
625596
625597
625598
625599
625600
625601
625602
625603
625604
625605
625606
625607
625608
625609
625610
625611
625612
625613
625614
625615
625616
625617
625618
625619
625620
625621
625622
625623
625624
625625
625626
625627
625628
625629
625630
625631
625632
625633
625634
625635

626664
626665
626666
626667
626668
626669
626670
626671
626672
626673
626674
626675
626676
626677
626678
626679
626680
626681
626682
626683
626684
626685
626686
626687
626688
626689
626690
626691
626692
626693
626694
626695
626696
626697
626698
626699
626700
626701
626702
626703
626704
626705
626706
626707
626708
626709
626710
626711
626712
626713
626714
626715
626716
626717
626718
626719
626720
626721
626722
626723
626724
626725
626726
626727
626728
626729
626730
626731
626732
626733
626734
626735
626736
626737
626738
626739
626740
626741
626742
626743
626744
626745
626746
626747
626748
626749
626750
626751
626752
626753
626754
626755
626756
626757
626758
626759
626760
626761
626762
626763
626764
626765
626766
626767
626768
626769
626770
626771
626772
626773
626774
626775
626776
626777
626778
626779
626780
626781
626782
626783
626784
626785
626786
626787
626788
626789
626790
626791
626792
626793
626794
626795
626796
626797
626798
626799
626800
626801
626802
626803
626804
626805
626806

In [8]:
import requests
import sqlite3
import datetime
from bs4 import BeautifulSoup
from soynlp.normalizer import *
con = sqlite3.connect("ajou_data.db")
cur = con.cursor()
cur.execute('select * from data')
data = cur.fetchall()
con.commit()
con.close()

In [10]:
len(data)

57544